crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -511,6 +511,62 @@ impl<'w, 's, 'a> EntityCommands<'w, 's, 'a> {
         self
     }
 
+    /// Moves a [`Bundle`] from the entity to another.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use bevy_ecs::prelude::*;
+    /// #
+    /// # #[derive(Component)]
+    /// # struct Dummy;
+    /// #
+    /// # #[derive(Bundle)]
+    /// # struct Cargo {
+    /// #   _dummy: Dummy
+    /// # }
+    /// #
+    /// # #[derive(Component)]
+    /// # struct NextDestination { entity: Entity }
+    /// #
+    /// fn transport(mut commands: Commands, query: Query<(Entity, &NextDestination)>) {
+    ///     for (source, NextDestination { entity: target }) in query.iter() {
+    ///         commands.entity(source).move_bundle::<Cargo>(*target);
+    ///     }
+    /// }
+    /// ```
+    pub fn move_bundle<T: Bundle>(&mut self, target: Entity) -> &mut Self {
+        self.commands.add(MoveBundle::<T> {
+            source: self.entity,
+            target,
+            _bundle: PhantomData,
+        });
+        self
+    }
+
+    /// Moves a [`Component`] from the entity to another.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use bevy_ecs::prelude::*;
+    /// #
+    /// # #[derive(Component)]
+    /// # struct Charge;
+    /// #
+    /// # #[derive(Component)]
+    /// # struct ConnectedTo { entity: Entity }
+    /// #
+    /// fn conduction(mut commands: Commands, query: Query<(Entity, &ConnectedTo), With<Charge>>) {
+    ///     for (source, ConnectedTo { entity: target }) in query.iter() {
+    ///         commands.entity(source).r#move::<Charge>(*target);
+    ///     }
+    /// }
+    /// ```
+    pub fn r#move<C: Component>(&mut self, target: Entity) -> &mut Self {
```

Comment:
What should we do about `r#move`?

Reply:
One possible side step here to name these `move_to` and `move_bundle_to`. 

https://github.com/bevyengine/bevy/pull/3820 could then be `clone_to` and `clone_bundle_to`.

Reply:
This would also allow for the addition of `move_from`, which would take the `source` `Entity` as an argument.

Reply:
IMO `move_to` is probably better. And then do the same with `clone_to`.

Reply:
@alice-i-cecile Will do, I'm also happy with `move_onto`, if you'd prefer that?